src/super_gradients/training/transforms/transforms.py

```diff
@@ -487,18 +487,31 @@ class DetectionRandomAffine(DetectionTransform):
      shear: (Union[tuple, float]) degrees for random shear, when float the random values are drawn uniformly
         from (shear, shear)
 
-    enable: (bool) whether to apply the below transform at all.
+     enable: (bool) whether to apply the below transform at all.
+
+     filter_box_candidates: (bool) whether to filter out transformed bboxes by edge size, area ratio, and aspect ratio (default=False).
+
+     wh_thr: (float) edge size threshold when filter_box_candidates = True (default=2)
```

Comment:
write that boxes *smaller* than threshold will be filtered out

Reply:
all thresholds clarified in docs https://github.com/Deci-AI/super-gradients/commit/8bdf12438e74d3efb245b92baa1ff9ffef3c7306